src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_params.py

```diff
@@ -41,6 +40,8 @@
 
 register_cli_argument('resource list', 'name', resource_name_type)
 register_cli_argument('resource move', 'ids', nargs='+')
+register_cli_argument('resource invoke-action', 'action', help='The action that will be invoked')
```

Comment:
What action? Http action? Should we elaborate in the help?

Reply:
How would someone know what actions are possible on a given resource?